CliWrap/CliWrap.csproj

```diff
@@ -15,4 +15,16 @@
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
   </PropertyGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
```

Comment:
What's the reason for adding these packages?

Reply:
It didn't compile with .NET Standard and Core, but it doesn't work anyway so I'll try to remove them.